src/Resources/skeleton/authenticator/login_form.tpl.php

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Log in!{% endblock %}
+
+{% block body %}
+<form method="post">
+    {% if error %}
+        <div class="alert alert-danger">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+    {% endif %}
+
+    <h1 class="h3 mb-3 font-weight-normal">Please sign in</h1>
+    <label for="inputEmail" class="sr-only">Email address</label>
+    <input type="email" value="{{ last_username }}" name="<?= $username_field; ?>" id="inputEmail" class="form-control" placeholder="Email address" required autofocus>
```

Comment:
Ah.... this should only be `type="email"` if their "username" field contains the term `email` I think. I chose `username` for my User class, and so this field should not have been type="email".

Also, though annoying, we should probably make the `id=""` and label's `for=""` a bit dynamic - based on the actual identity field name.

AND... of course - the label's text - `Email address` - should be dynamic - probably we can turn their propery name into title-case words - e.g. `emailAddress` -> `Email Address` or `email` to just `Email`

Reply:
ok you're absolutely right